<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teBricks</class>
 <widget class="QWidget" name="teBricks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frBrickTypes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Brick type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="brickType"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frBrickWidth">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Brick width</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="brickWidth">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frBrickWidth_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Brick height</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="brickHeight">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frBrickWidth_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Brick depth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="brickDepth">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QFrame" name="frBrickWidth_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Brick bevel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="brickBevel">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frBrickWidth_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Mortar size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="mortarSize">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QFrame" name="frBrickWidth_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Brick offset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="brickOffset">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="ReTextureAvatar" name="brickTexture">
     <property name="caption">
      <string>Brick</string>
     </property>
     <property name="breadcrumb">
      <string>Brick Tex.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="ReTextureAvatar" name="mortarTexture">
     <property name="caption">
      <string>Mortar</string>
     </property>
     <property name="breadcrumb">
      <string>Mortar Tex.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="ReTextureAvatar" name="brickModulation">
     <property name="caption">
      <string>Modulation</string>
     </property>
     <property name="breadcrumb">
      <string>Modulation Tex.</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="Re3DMappingEditor" name="mapping3D"/>
   </item>
   <item row="8" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>176</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frBrickPreview">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="lyNoiseResult">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Preview</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="noiseResult">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>144</width>
             <height>144</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>144</width>
             <height>144</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="msgRenderingPreview">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Rendering preview...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Re3DMappingEditor</class>
   <extends>QWidget</extends>
   <header>Re3DMappingEditor.h</header>
  </customwidget>
  <customwidget>
   <class>ReTextureAvatar</class>
   <extends>QWidget</extends>
   <header>ReTextureAvatar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
